.gitattributes

```diff
@@ -0,0 +1,3 @@
+Dockerfile* linguist-language=Dockerfile
+vendor.mod linguist-language=Go-Module
+vendor.sum linguist-language=Go-Checksums
```

Comment:
@errordeveloper Just stylish stuff you can add to match to proper language using [gitattributes](https://github.com/github/linguist/blob/master/docs/overrides.md#using-gitattributes):

* https://github.com/crazy-max/docker-cli/blob/vendor-gomod/vendor.mod
* https://github.com/crazy-max/docker-cli/blob/vendor-gomod/vendor.sum

Reply:
nice!